philips-hue-adapter.js

```diff
@@ -130,14 +130,14 @@ class PhilipsHueProperty extends Property {
         const levelProp = this.device.findProperty('level');
 
         if ((colorModeProp && colorModeProp.value === 'color') ||
-            (!colorModeProp && colorProp)) {
+          (!colorModeProp && colorProp)) {
```

Comment:
Not sure what happened with the spacing on these `if` blocks.

Reply:
Done